backend/tests/mongo_utils.js

```diff
@@ -6,30 +6,22 @@ let mongoServer
 
 before(async () => {
   mongoServer = new MongoMemoryServer()
-  // mongoServer
-  //   .getConnectionString()
-  //   .then(mongoUri => {
-  //     return mongoose.connect(
-  //       mongoUri,
-  //       {},
-  //       err => {
-  //         if (err) {
-  //           done(err)
-  //         }
-  //       }
-  //     )
-  //   })
-  //   .then(() => done())
+  mongoServer
+    .getConnectionString()
+    .then(mongoUri => {
+      return mongoose.connect(mongoUri, {}, err => {
+        if (err) {
+          done(err)
+        }
+      })
+    })
+    .then(() => done())
```

Comment:
is uncommenting this what fixed it? This was the old version that I tried to rewrite with async/await with lines 19-22. Tbh idk how this even runs since `done` isn't defined anymore.

Reply:
Noo it fixed one of them where mongo wasn't returning the _id value of an object. When i changed the leads key to `director` instead of `lead` tho a bunch of the tests passed

Reply:
ok lets attempt to get rid of it again? if that doesn't work we can leave it, but I don't really understand how leaving this isn't causing everything to break lol